src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/ConfigCommands/ConfigRunners.cs

```diff
@@ -100,6 +115,19 @@ public static ISettings GetSettingsFromDirectory(string directory)
         }
 
         /// <summary>
+        /// Creates a settings object utilizing a NuGet configuration file path.
+        /// </summary>
+        /// <param name="filePath"></param>
+        /// <returns></returns>
+        public static ISettings GetSettingsFromFile(string filePath)
+        {
+            var directory = Path.GetDirectoryName(filePath);
+            return NuGet.Configuration.Settings.LoadDefaultSettings(
```

Comment:
This will only load the settings from the specified config file, right?
When user specifies `--configfile`  in `set` command, what would happen if the config file path doesn't exist? (e.g. user enters a wrong path)
Will we show error message saying the config file doesn't exist? Thanks!

Reply:
Yes, you get this message: `error: File 'filepath' does not exist.`